src/config.ts

```diff
@@ -157,6 +157,7 @@ const DEFAULTS = o({
     "homepages": [],
     "hintchars": "hjklasdfgyuiopqwertnmzxcvb",
 	"hintfiltermode": "simple",   // "simple", "vimperator", "vimperator-reflow"
+    "allowautofocus": false,
```

Comment:
`false` should be a string, "false". Set cannot set boolean values.

I think it should be "true" by default, as that is the standard behaviour in Firefox (and Vimperator?).